atomic_physics/tests/test_utils.py

```diff
@@ -1,275 +1,205 @@
+        # All pi shifts should be zero since there are no pi spectator transitions.
+        acz_pi_0 = ac_zeeman_shift_for_state(ion, 0, rf_drive_pi)
+        acz_pi_1 = ac_zeeman_shift_for_state(ion, 1, rf_drive_pi)
+        acz_pi_transition = ac_zeeman_shift_for_transition(ion, [0, 1], rf_drive_pi)
+        self.assertEqual([acz_pi_0, acz_pi_1, acz_pi_transition], [0.0, 0.0, 0.0])
 
-        np.testing.assert_allclose(
-            shift,
-            ac_zeeman_shift_for_state(
-                atom=ion,
-                state=victim_state_index,
-                drive=RFDrive(
-                    frequency=w_rf,
-                    amplitude=rf_amplitude,
-                    polarization=polarization.SIGMA_MINUS_POLARIZATION,
-                ),
-            ),
+        # All sigma minus shifts should be zero since there are no sigma minus
+        # spectator transitions.
+        acz_sigma_m_0 = ac_zeeman_shift_for_state(ion, 0, rf_drive_minus)
+        acz_sigma_m_1 = ac_zeeman_shift_for_state(ion, 1, rf_drive_minus)
+        acz_sigma_m_transition = ac_zeeman_shift_for_transition(
+            ion, [0, 1], rf_drive_minus
+        )
+        self.assertEqual(
+            [acz_sigma_m_0, acz_sigma_m_1, acz_sigma_m_transition], [0.0, 0.0, 0.0]
         )
 
-        # sigma+ shift
-        spectator_state_index = ion.get_state_for_F(level=ca43.S12, F=4, M_F=0)
-        Omega = ion.get_rabi_m1(
-            lower=spectator_state_index,
-            upper=victim_state_index,
-            amplitude=rf_amplitude,
+        acz_sigma_p_0 = ac_zeeman_shift_for_state(ion, 0, rf_drive_plus)
+        acz_sigma_p_1 = ac_zeeman_shift_for_state(ion, 1, rf_drive_plus)
+        acz_sigma_p_transition = ac_zeeman_shift_for_transition(
+            ion, [0, 1], rf_drive_plus
         )
-        w_spectator = ion.get_transition_frequency_for_states(
-            states=(spectator_state_index, victim_state_index)
+        # The higher energy state moves down in energy by the expected shift and
+        # the lower energy state moves up in energy by the expected shift since
+        # the RF frequency is higher than the transition frequency. The frequency
+        # of the transition decreases by double the expected shift.
+        self.assertAlmostEqual(acz_sigma_p_0, -expected_shift_abs, delta=1e-8)
+        self.assertAlmostEqual(acz_sigma_p_1, expected_shift_abs, delta=1e-8)
+        self.assertAlmostEqual(
+            acz_sigma_p_transition, -2 * expected_shift_abs, delta=1e-8
         )
-        shift = ac_zeeman(Omega, w_spectator, w_rf)
 
-        np.testing.assert_allclose(
-            shift,
-            ac_zeeman_shift_for_state(
-                atom=ion,
-                state=victim_state_index,
-                drive=RFDrive(
-                    frequency=w_rf,
-                    amplitude=rf_amplitude,
-                    polarization=polarization.SIGMA_PLUS_POLARIZATION,
+    def test_acz_sideband_ca43(self):

```

Comment:
Again, I'd be inclined to just call this `test_acz_ca43` or something and not reference a sideband.

Reply:
Well, there are two Ca43 tests: one with qubit freq +/- 3 MHz, and one with trap RF. How would you propose to differentiate and also explain why we have both the +3MHz and -3 MHz tones? I also think it's nice to keep the same notation as your thesis.

Reply:
okay, fine